diffkemp/cli.py

```diff
@@ -105,6 +105,10 @@ def make_argument_parser():
     compare_ap.add_argument("--report-stat",
                             help="report statistics of the analysis",
                             action="store_true")
+    compare_ap.add_argument("--extended-stat",
+                            help="compute extended statistics that are more "
+                                 "expensive to compute (e.g. line count). ",
```

Comment:
Maybe rephrase this?
```suggestion
                            help="report detailed statistics about the analysis "
                                 "(may be more expensive to compute). ",
```